src/rws_rapid.cpp

```diff
@@ -248,9 +248,9 @@ std::ostream& operator<<(std::ostream& os, ConfData const& robconf)
 {
     return os
         << "ConfData("
-        << "cf1=" << robconf.cf1.value // quadrent number for axis 1
-        << "cf4=" << robconf.cf4.value // quadrent number for axis 4
-        << "cf6=" << robconf.cf6.value // quadrent number for axis 6
+        << ".cf1=" << robconf.cf1.value // quadrent number for axis 1
+        << ", .cf4=" << robconf.cf4.value // quadrent number for axis 4
+        << ", .cf6=" << robconf.cf6.value // quadrent number for axis 6
```

Comment:
Quadrant?

Reply:
That's true. And I lost `.cfx` value I think.

Reply:
Fixed